### DISCUSSION TITLE
Are there realtime limits or other rate limits?

### DISCUSSION THREAD

**Discussion by richcorbs** [supabase/supabase#2510] [Questions] [answered]
I created a small scrum pointing app that uses subscriptions. When four different people joined/subscribed the realtime failed for the third and fourth people. 

![subscription-failure](https://user-images.githubusercontent.com/437167/126798401-b07bbaae-d851-415e-9648-96b9f7539189.png)

Has anyone else seen this?

When I simulate this using four different browsers from my local system I don't have the same issue.

Is it a timing thing? Are Supabase connections somehow rate-limited on the server?

**Comment by inian** [comment, marked as answer]
More context here @richcorbs https://github.com/supabase/supabase/issues/2096
